.sequelizerc

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+require('dotenv').config();
```

Comment:
Why are you requiring dotenv here?

Reply:
It was generated by my sequelize-cli. but I think it's not necessary to be there